code/modules/fishing/fishing_gear.dm

```diff
@@ -350,3 +350,341 @@ TYPEINFO(/obj/item/fish_portal)
 	item_state = "aquarium"
 	slots = 6
 	can_hold = 	list(/obj/item/reagent_containers/food/fish)
+
+TYPEINFO(/obj/item/syndie_fishing_rod)
+	mats = list("MET-3"=15, "WOOD"=5)
+
+/obj/item/syndie_fishing_rod
+	name = "\improper Glaucus fishing rod"
+	desc = "A high grade tactical fishing rod, completely impractical for reeling in bass."
+	icon = 'icons/obj/items/fishing_gear.dmi'
+	icon_state = "syndie_fishing_rod-inactive"
+	inhand_image_icon = 'icons/mob/inhand/hand_fishing.dmi'
+	item_state = "syndie_fishing_rod-inactive"
+	hit_type = DAMAGE_STAB
+	flags = FPRINT | TABLEPASS | USEDELAY
+	w_class = W_CLASS_NORMAL
+	force = 10
+	throwforce = 5
+	throw_speed = 1
+	throw_range = 5
+	contraband = 4
+	is_syndicate = TRUE
+	var/obj/item/implant/syndie_lure/lure = null
+	// delay between tossing or reeling or etc
+	var/usage_cooldown = 0.8 SECONDS
+	// time per step to reel/filet a mob
+	var/syndie_fishing_speed = 0.7 SECONDS
+	// cooldown after throwing a hooked target around
+	var/yank_cooldown = 6 SECONDS
+	// how far you throw when yanking them
+	var/yank_range = 4
+	// how far the line can stretch
+	var/line_length = 9
+	// true if the rod is currently ""fishing"", false if it isnt
+	var/is_fishing = FALSE
+	HELP_MESSAGE_OVERRIDE({"The Glaucus starts with 7 damage on a melee reel, but builds up 3 additional damage on each ranged reel. If this reaches <b>25 damage</b>, or 6 ranged reels before a melee reel, the target will be stunned when damaged."})
+
+	New()
+		..()
+		src.lure = new (src)
+		src.lure.rod = src
+
+	get_desc(dist)
+		..()
+		if (dist < 1) // on our tile or our person
+			if (.) // we're returning something
+				. += " " // add a space
+			if (!src.lure)
+				src.lure = new (src)
+				src.lure.rod = src
+			. += "There is \a [src.lure.name] presented as bait."
+
+	attackby(obj/item/I, mob/user)
+		if (!src.lure)
+			src.lure = new (src)
+			src.lure.rod = src
+		if (src.lure.loc == src)
+			boutput(user, "You scan \the [I.name] onto \the [src.name]'s holographic bait projector.")
+			src.lure.name = I.name
+			src.lure.real_name = I.name
+			src.lure.desc = I.desc
+			src.lure.real_desc = I.desc
+			src.lure.appearance = I
+			src.lure.set_dir(I.dir)
+			src.lure.overlay_refs = I.overlay_refs?.Copy()
+			src.lure.plane = initial(src.lure.plane)
+			src.lure.layer = initial(src.lure.layer)
+			src.lure.tooltip_rebuild = 1
+			tooltip_rebuild = 1
+		else
+			boutput(user, "You can't change the bait while the line is out!")
+		return
+
+	attack_self(mob/user)
+		. = ..()
+		if (!src.lure)
+			src.lure = new (src)
+			src.lure.rod = src
+		if (src.lure.loc == src)
+			boutput(user, "You clean \the [src.name]'s holographic bait projector.")
+			src.lure.clean_forensic()
+		else
+			if(!src.lure.owner)
+				if(BOUNDS_DIST(src.lure,src) == 0)
+					src.is_fishing = FALSE
+					src.UpdateIcon()
+					user.update_inhands()
+					src.lure.set_loc(src)
+				else
+					step_towards(src.lure, src)
```

Comment:
This should check that the lure is on a turf, currently it runtimes if you try to reel it in while it's in a disposal pipe etc.